ResidentContactApi.Tests/V1/Gateways/ResidentGatewayTests.cs

```diff
@@ -22,6 +23,9 @@ public class ResidentGatewayTests : DatabaseTests
         [SetUp]
         public void Setup()
         {
+            _fixture.Behaviors.OfType<ThrowingRecursionBehavior>().ToList()
```

Comment:
Could you explain what this is doing? Why are we removing recursive behaviour?

Reply:
If you have two objects which both reference each other then when fixture tries to create it, it errors because there's an infinite loop of nested objects to create. for example ```ContactDetails { 
public Resident resident 
} ``` and ``` Resident {
List<ContactDetails> contacts
} ```

Reply:
I see. Okay.. are we letting it create an infinite loop or telling it how many to make before breaking the loop?

Reply:
Saying how many to do before breaking the loop

Reply:
👍